tests/cli/test_suite.py

```diff
@@ -1146,14 +1148,12 @@ def test_suite_edit_with_non_existent_datasource_shows_helpful_error_message(
     - NOT open jupyter
     """
     context = empty_data_context_stats_enabled
-    monkeypatch.chdir(os.path.dirname(context.root_directory))
+    monkeypatch.chdir(os.path.dirname(context.root_directory))  # noqa: PTH120
 
     expectation_suite_name: str = "test_suite_name"
 
     # noinspection PyUnusedLocal
-    suite: ExpectationSuite = context.add_expectation_suite(
```

Comment:
Do we not want to save the suite returned from add_expectation_suite in these 2 places?